Src/Newtonsoft.Json/Utilities/DynamicProxyMetaObject.cs

```diff
@@ -176,7 +176,7 @@ public override DynamicMetaObject BindDeleteIndex(DeleteIndexBinder binder, Dyna
 
         private delegate DynamicMetaObject Fallback(DynamicMetaObject errorSuggestion);
 
-        private static readonly Expression[] NoArgs = new Expression[0];
+        private static Expression[] NoArgs => CollectionUtils.ArrayEmpty<Expression>();
```

Comment:
Because this is a generic class this allocated an identical field for every `T` the class was used with. Changing to an inlinable property reduces this.